dev-client/src/components/buttons/DeleteButton.tsx

```diff
@@ -36,6 +36,7 @@ export default function DeleteButton({label, ...props}: Props) {
       p={0}
       pt="10px"
       startIcon={<Icon name="delete" color="error.main" size="md" mr={2} />}
+      isDisabled={props?.disabled ? props.disabled : undefined}
```

Comment:
Suggestion/question: This is equivalent to just `props?.disabled`, right?

Reply:
That results in a TypeScript error:
```
Type 'boolean | null | undefined' is not assignable to type 'boolean | undefined'.
  Type 'null' is not assignable to type 'boolean | undefined'.ts(2322)
```